rnd/autogpt_server/autogpt_server/util/json.py

```diff
@@ -11,5 +11,4 @@ def dumps(data) -> str:
     return json.dumps(jsonable_encoder(data))
 
 
-def loads(data) -> dict:
-    return json.loads(data)
+loads = json.loads
```

Comment:
? wtf

Reply:
Yeah it's not pretty, this module is imported as `from utils import json` and then used as `json.loads(..)`, `json.dumps(..)`. Not my idea.